portal/templates/portal_wrapper.html

```diff
@@ -79,6 +79,11 @@
                             <li><a href="{{PORTAL}}/scheduled_jobs">{{ _("Scheduled Jobs") }}</a></li>
                             <li><a href="{{PORTAL}}/settings">{{ _("Settings") }}</a></li>
                             {% endif %}
+                            <!-- fixme use the new 'analytics' intervention's 'link_url' -->
+                            <!-- fixme consider intervention.display_for_user(user) ? -->
+                            {% if user and (user.has_role(ROLE.STAFF.value) or user.has_role(ROLE.STAFF_ADMIN.value)  or user.has_role(ROLE.ANALYST.value) or user.has_role(ADMIN.value)) %}
```

Comment:
Missing `ROLE` here...
```suggestion
                            {% if user and (user.has_role(ROLE.STAFF.value) or user.has_role(ROLE.STAFF_ADMIN.value)  or user.has_role(ROLE.ANALYST.value) or user.has_role(ROLE.ADMIN.value)) %}
```

Reply:
This might be a little more succinct...
```suggestion
                            {% if user and (any((user.has_role(r.value) for r in (ROLE.ADMIN, ROLE.ANALYST, ROLE.STAFF_ADMIN, ROLE.STAFF)))) %}
```